apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -380,8 +380,37 @@ type ResourceIdentifier struct {
 	// Namespace is the namespace of the resource. Empty if the resource is cluster scoped.
 	// +optional
 	Namespace string `json:"namespace,omitempty"`
+
+	// Envelope identifies the envelope objects that contains this resource.
+	// +optional
+	Envelope EnvelopeIdentifier `json:"bundleIdentifier,omitempty"`
 }
 
+// EnvelopeIdentifier identifies the bundle objects that contains the selected resource.
```

Comment:
Use "envelope" instead of "bundle" everywhere if you decide to use envelope. That means in the future you will add ClusterResourceEnvelope, ResourceEnvelope if needed.

s/the bundle objects/the envelope object/